docs/technologies/index.md

```diff
@@ -0,0 +1,83 @@
+# Technologies List
+
+Tools and technologies being used by UKHO engineering teams. Investigation of these technologies will have a sponsor and will either be investigated within a team (perhaps through an agile Spike), during a JFDI activity or by the architecture practice.
```

Comment:
Can we lose these lines as it isn't really true!

Reply:
Which bit's not true?

Reply:
In particular `These lists have been agreed and approved by the Lead Engineer Forum and the Architecture Team and will be reviewed on a regular basis with the latest version published here.` as it _clearly_ isn't true else the list would be up to date/accurate. And are the Architecture team involved with this list?

Thinking about it more, I am pushing back against this list being part of governance/process. It should just be a list of technologies for awareness sake.

The existence of this list hasn't stopped us from using different or new technologies, it is just another thing we haven't done and then something to feel bad about!

Reply:
Marc, are you saying that there should be no governance (i.e. that each team should use whatever they want)?  Or just that the list doesn't form part of that (I'm not sure how that would work - people need to know when they are free to use something)?

Reply:
I don't think this list should form a part of that. 

There has always been a difference between what the horizon technology list says and what we are actually using/looking at(prime example; all the stuff Ant has just added)

Having a list that we _know_ is behind/out of date/not well kept as part of a governance/process seems like a recipe to create frustration and get in the way.

Reply:
> It should just be a list of technologies for awareness sake.

I'm leaning towards agreeing with this to be honest. I think it should be for visibility so that technology choices can be spread, shared and challenged.

I do think that we should be more aware of the existence of the list though. It comes up for quarterly review at the leads meeting and that always ends up in agreeing that it's useless because it's outdated.

I'd like to make this more visible to teams including the architecture practice. Delivery teams can add the tech they are using so that we can avoid silo-ing.

Reply:
I think the horizon list is probably more trouble that it's worth.

I do, however, think that our future boss will expect us to be better governed in terms of introducing technologies than we currently are.  Is everyone really checking the licenses properly, making sure we're not stumbling into an "open wash" trap, etc?  Are all teams always going to act in accordance with the long-term interests of the UKHO instead of using the shiniest option?  A good example is the debate around client frameworks. Publishing a list for visibility doesn't really cut it IMO.

I think this should go on hold, pending further discussions.  Kari seems keen on autonomy at the moment, so I will see what she thinks.

Reply:
A way forward for technology governance is on Carl's list that he wants to sort out.  I therefore suggest we don't commit this change pending input from @thealmguy .